### PR TITLE
take request entitytoken into account before attempting to use static

### PR DESCRIPTION
**Pull request by toddbellMSFT**
and fix some warnings along the way

### CODE REVIEW

**Comment by pgilmorepf** [comment]
Nevermind, a (thankfully) minor aspect of the data-isolation fix was missed, or somehow lost from the earlier fix.

File: targets/csharp/UnittestRunner/UUnitTestRunner.cs [hunk 1]
```diff
@@ -5,12 +5,11 @@
 using PlayFab.ClientModels;
 using System.Threading.Tasks;
 
+#pragma warning disable 0649, 0414
```
Comment (toddbellMSFT) [inline targets/csharp/UnittestRunner/UUnitTestRunner.cs]: If you can remove the _onCompleted, then we can remove the warning supressions (they are both for unused fields/ unused assignments)

Comment (pgilmorepf) [reply targets/csharp/UnittestRunner/UUnitTestRunner.cs]: There's other unused in this file as well, so I left them.

**Comment by pgilmorepf** [comment]
Manual test kick-off passed. Landing.

**Pull request merged**